LittleHelpBookData/little_help_book.py

```diff
@@ -0,0 +1,100 @@
+import os
+import sys
+from datetime import datetime
+import re
+import numpy as np
+import requests
+from bs4 import BeautifulSoup
+import pandas as pd
+from pymongo import MongoClient, errors
+from tqdm import tqdm
+
+_i = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+if _i not in sys.path:
+    # add parent directory to sys.path so utils module is accessible
+    sys.path.insert(0, _i)
+del _i  # clean up global name space
+from shared_code.utils import (
+    check_similarity, locate_potential_duplicate,
+    insert_services, client
+)
+from shared_code.base_scraper import BaseScraper
+
+
+class LHB_Scraper(BaseScraper):
+
+    def scrape_updated_date(self):
+
+        resp = super().scrape_updated_date()
+        soup = BeautifulSoup(resp, 'html.parser')
+        date_string = soup.find('relative-time', class_='no-wrap')
+        date_string = date_string.attrs['datetime']
+        date_string = re.search(
+            r'(19|20)\d\d[- /.](0[1-9]|1[012])[- /.](0[1-9]|[12][0-9]|3[01])',
+            date_string
+        ).group(0)
+        scraped_update_date = datetime.strptime(
+            date_string, '%Y-%m-%d'
+        )
+        return scraped_update_date.date()
+
+
+    def grab_data(self):
+
+        '''Subcategory would be serviceSummary, Servicename would be name, PhoneNumber would be phone,
```

Comment:
I see some entries with blank subcategory in which case we should use main category as the service summary